### DISCUSSION TITLE
How to set the bearer token dynamically in graphql-spring-boot playground

### DISCUSSION THREAD

**Discussion by skesani** [graphql-java-kickstart/graphql-spring-boot#928] [Q&A]
Hey All,

I am using playground embedded by accessing at root ` /playground`, I am checking for a solution on how to set the bearer token based dynamically, current documentation shows,  we can set the static token in .yml file like below, however i wanted to set the token programmatically since the token changes based on the user roles and when i navigate to ` /playground` the token should set like below image, please let me know if this is possible? if so please provide a code sample for it. Thanks.
  
```
headers:
      Authorization: "Bearer <your-token>"
```

![image](https://user-images.githubusercontent.com/2635028/225638976-64ee25c2-bac2-40ea-af0c-159987ded592.png)

https://github.com/graphql-java-kickstart/graphql-spring-boot#enable-graphql-playground

**Comment by BlasiusSecundus** [comment]
Yes, it will work. The variables set in the "HTTP headers" tab will be sent to the server as the payload of the `connection_init` message.
